providers/register/provider_openstack.go

```diff
@@ -0,0 +1,21 @@
+// +build !no_openstack_provider
```

Comment:
Sorry, we ended up flipping this around. Can you change the build tag to `openstack_provider`

Reply:
Done